src/dom.js

```diff
@@ -573,7 +578,7 @@ export function scopedQuerySelectorAll(root, selector) {
 /**
  * Returns element data-param- attributes as url parameters key-value pairs.
  * e.g. data-param-some-attr=value -> {someAttr: value}.
- * @param {!Element} element
+ * @param {!HTMLElement} element
```

Comment:
For my own understanding, what's the difference in these types?

Reply:
HTMLElement extends Element which extends the interface Node.  Element doesn't have custom element callbacks whereas HTMLElement does

Reply:
Thanks for clarifying!